components/style/properties/gecko.mako.rs

```diff
@@ -1338,17 +1361,7 @@ ${impl_style_struct(style_struct)}
 ${define_ffi_struct_accessor(style_struct)}
 % endfor
 
-lazy_static! {
-    pub static ref INITIAL_GECKO_VALUES: ComputedValues = ComputedValues {
-        % for style_struct in data.style_structs:
-           ${style_struct.ident}: style_structs::${style_struct.name}::initial(),
-        % endfor
-        custom_properties: None,
-        shareable: true,
-        writing_mode: WritingMode::empty(),
-        root_font_size: longhands::font_size::get_initial_value(),
-    };
-}
+static mut INITIAL_GECKO_VALUES: *mut ComputedValues = 0 as *mut ComputedValues;
 
```

Comment:
Why should it scare you? It's the *minimum* you want. `static mut` is less than the minimum.
Relaxed ordering + `assert!` instead of `debug_assert!` is all you need if you want 100% safety.